README.rst

```diff
@@ -242,6 +242,58 @@ Run qa tests:
 
     ./run-qa-checks
 
+Quickstart Guide
+----------------
+
+This module works by periodically collecting the network topology
+graph data from the supported softwares (NetJSON NetworkGraph, OLSR (jsoninfo/txtinfo),
+batman-adv (jsondoc/txtinfo), BMX6 (q6m), CNML 1.0, OpenVPN). The data has to be
```

Comment:
@pandafy never duplicate information which is referred elsewhere, always follow the DRY principle